src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -34,6 +34,13 @@
   "Schema defining the supported values for the JRuby CompileMode setting."
   (apply schema/enum supported-jruby-compile-modes))
 
+(def supported-jruby-profiling-modes
```

Comment:
Not seeing this used elsewhere, is there a a reason to have it broken out of the schema?

Reply:
i copied what had been done for the `supported-jruby-compile-modes`

Reply:
It looks like Chris was using that set in an integration test.

https://github.com/puppetlabs/jruby-utils/commit/9d44cfed213c387fabb041433cf6de7b8946060e#diff-78532c5c3713f4d2fe00eda939a3e93fR19

Reply:
(unit)

Reply:
i can inline these if that still makes sense.